### DISCUSSION TITLE
Next release date

### DISCUSSION THREAD

**Discussion by janatjak** [thecodingmachine/graphqlite#643] [General]
Hello @oojacoboo 

Can i have a question please? When is the next feature release scheduled please?

I looking forward to the new features (Symfony 7 and param doc types).

Thanks!

Btw: You're doing a great job.

**Comment by oojacoboo** [comment]
Now would actually be a good time to do a release.  It'd be nice to find a release manager for this lib.  Any interest?

**Reply by aszenz** [reply]
I'm interested in adding symfony 7 support for the bundle, does that also need changes in core graphqlite

**Reply by oojacoboo** [reply]
@aszenz nothing is needed here for the Symfony bundle.  That's  a separate lib - submit a PR for it as needed.
